src/MediaManagerBundle/Bridge/Chameleon/DataAccess/MediaItemDataAccess.php

```diff
@@ -279,38 +279,62 @@ private function addSearchRestrictions(
         array &$paramTypes
     ) {
         $searchTerm = $mediaManagerListRequest->getSearchTerm();
-        if (null === $searchTerm || '' === trim($searchTerm)) {
+        $searchTerm = trim($searchTerm);
+        if (null === $searchTerm || '' === $searchTerm) {
             return;
         }
 
         $language = $this->languageService->getLanguage($languageId);
-        $parts = array();
+        $parts = [];
+
+        $parts[] = '`id` = :fullTerm';
+        $parts[] = '`cmsident` = :fullTerm';
 
-        $parts[] = '`id` LIKE :fullTerm';
-        $parts[] = '`cmsident` LIKE :fullTerm';
         $params['fullTerm'] = $searchTerm;
         $paramTypes['fullTerm'] = \PDO::PARAM_STR;
 
         $terms = preg_split(',[\.\/\\-\s],', $searchTerm);
 
-        $descriptionFieldName = $this->fieldTranslationUtil->getTranslatedFieldName(
+        $descriptionFieldName = 'description';
+        $descriptionFieldNameTranslated = $this->fieldTranslationUtil->getTranslatedFieldName(
             'cms_media',
-            'description',
+            $descriptionFieldName,
             $language
         );
 
-        $pathFieldName = $this->fieldTranslationUtil->getTranslatedFieldName('cms_media', 'path', $language);
+        $pathFieldName = 'path';
+        $pathFieldNameTranslated = $this->fieldTranslationUtil->getTranslatedFieldName('cms_media', $pathFieldName, $language);
         $tagNameFieldName = $this->fieldTranslationUtil->getTranslatedFieldName('cms_tags', 'name', $language);
 
+        $descriptionSearchQueryParts = [];
         $i = 0;
         foreach ($terms as $term) {
             ++$i;
-            $parts[] = $this->databaseConnection->quoteIdentifier($descriptionFieldName).' LIKE :term'.(string) $i;
-            $parts[] = $this->databaseConnection->quoteIdentifier($pathFieldName).' LIKE :term'.(string) $i;
+
+            $descriptionSearchQueryParts[$descriptionFieldNameTranslated][] = $this->databaseConnection->quoteIdentifier($descriptionFieldNameTranslated).' LIKE :term'.(string) $i;
+
+            if ($descriptionFieldNameTranslated !== $descriptionFieldName) {
```

Comment:
I wouldn't expect the system to search in other languages as the currently active language. Why would you like to do this?

Reply:
It's not some language... it's the active language and the base language, because in 99% of the cases the images where uploaded in the base language and not in a different language... and it would massively disturb users if the uploaded file is gone after they switched the language. The old media manager bahaved like this i'm pretty sure, because there we used the normal list manager and the list manager uses a fallback strategy to the base language.

Reply:
To be clear. The Names shown for the images have a fallback to the base language. So you will see in german the english base language name, but you would not be able to search for them without this change in the active language.